app/src/main/java/com/asfoundation/wallet/ui/common/WalletTextFieldView.kt

```diff
@@ -73,35 +74,43 @@ class WalletTextFieldView : FrameLayout {
   private fun applyType() {
     when (type) {
       Type.FILLED -> {
+        views.textInputEditText.setReadOnly(value = false, inputType = InputType.TYPE_CLASS_TEXT)
         views.textInputLayout.boxBackgroundColor = color
         views.textInputLayout.boxStrokeColor =
             ContextCompat.getColor(this.context, R.color.transparent)
         views.textInputLayout.boxStrokeWidth = 0
         views.textInputLayout.endIconMode = END_ICON_NONE
-        views.textInputEditText.inputType = InputType.TYPE_CLASS_TEXT
       }
       Type.OUTLINED -> {
-        views.textInputLayout.boxBackgroundColor =
-            ContextCompat.getColor(this.context, R.color.transparent)
+        views.textInputEditText.setReadOnly(value = false, inputType = InputType.TYPE_CLASS_TEXT)
+        views.textInputLayout.boxBackgroundColor = color
         views.textInputLayout.boxStrokeColor = color
```

Comment:
In the textInputLayout.boxStrokeColor, I think we want to put the gray color directly instead of the "color" variable. The color that comes from the context.obtainStyledAttributes might not be working